W-17855915-RTM-Agent-new-config (#768)
mulesoft/docs-runtime-manager: runtime-manager/modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

This directory contains all custom modules added to the agent (not included in the agent distribution).
* Updates the default agent truststore name from `truststore.jks` to `anypoint-truststore.jks` (for updates to versions 1.11.0 or 2.1.4 and later).

== Network Configuration for Runtime Manager Agent Version 2.7.0

> Before installing or updating to Runtime Manager agent Version 2.7.0, update your network configuration to add the following endpoint to your allowlist:

Reviewer: ```suggestion
Before installing or updating to Runtime Manager agent Version 2.7.0, update your network configuration to add this endpoint to your allowlist:
```
